examples/node-app-mssql/src/migrations/20191124213800_create_task_table.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Create table `tasks`.
+ *
+ * @param {Knex} knex
+ * @returns {Promise<any>}
+ */
+exports.up = function(knex) {
```

Comment:
Can we do `export function` syntax instead?

Reply:
Perhaps you'll need a stub file for this. cc @mesaugat

Reply:
> Can we do `export function` syntax instead?

To use `es6` syntax we need a transpiler(babel). Shall I add a `babel` config

Reply:
Yes, let's do.